spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+require 'factory_bot'
+
+RSpec.configure do |config|
```

Comment:
I prefer having these configurations inside one file (rails_helper). It gives you a full vision of what things are configurated  in your specs